pina/geometry/simplex.py

```diff
@@ -0,0 +1,232 @@
+import torch
+from .location import Location
+from pina.geometry import CartesianDomain
+from pina import LabelTensor
+from ..utils import check_consistency
+
+
+class SimplexDomain(Location):
+    """PINA implementation of a Simplex."""
+
+    def __init__(self, simplex_dict, labels, sample_surface=False):
```

Comment:
I would change the arguments in order to have only the mandatory `vertices` argument and the optional `sample_surface`. 
`vertices` can be:
- a `dict` in the form `{'x': [0.1, 3.2, 5], 'y': [7., 5., 0.01]}`
- a `LabelTensor` with the labels and any row with a vertex
It looks simpler also for the user imho!

P.S. be careful about the order of the points

Reply:
Yes agree, maybe the `LabelTensor` option is the simpler. So we remove the vertices name, and we instead directly pass the matrix. @benv123 Can you manage to do it?

Reply:
Yes I will work on it then commit when I'm done.